src/css/flags/overlay.less

```diff
@@ -5,4 +5,9 @@
     .jw-controls-right .jw-logo {
         display: none;
     }
+    &-sharing, &-related {
```

Comment:
This is currently true. I was initially going to do it that way, but after discussing with @robwalch, I concur that it's safer to limit the behavior to just the plugins for now, allowing something else to implement overlays slightly differently if the need arises. We could alternatively make it `&-plugin`, though, and apply that class to both.